make/Makefile.container.mk

```diff
@@ -78,4 +85,11 @@ else
 endif
 
 ## container-push: Pushes all container images to quay and docker hub.
-container-push: container-push-kiali-quay container-push-kiali-docker container-push-operator-quay
+container-push:
+# On x86_64 machine, push both kiali and operator images.
+ifeq ($(GOARCH),amd64)
+	make container-push-kiali-quay container-push-kiali-docker container-push-operator-quay
+# On other achitectures, only push kiali image.
+else
+	make container-push-kiali-quay container-push-kiali-docker
+endif
```

Comment:
Same as above. Do not call `make` directly. Do this instead:

```
ifeq ($(GOARCH),amd64)
container-push: container-push-kiali-quay container-push-kiali-docker container-push-operator-quay
else
container-push: container-push-kiali-quay container-push-kiali-docker
endif
```

Reply:
Done.